observability/logging/logging-6.0/log6x-opentelemetry-data-model.adoc

```diff
@@ -0,0 +1,381 @@
+|resource
+|container
+|structured metadata
+|
+
+|`k8s.pod.name`
+|resource
+|container
+|stream label
+|
+
+|`k8s.pod.uid`
+|resource
+|container
+|structured metadata
+|
+
+|`k8s.cronjob.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.daemonset.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.deployment.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.job.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.replicaset.name`
+|resource
+|container
+|structured metadata
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.statefulset.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`log.iostream`
+|log
+|container
+|structured metadata
+|
+
+|`k8s.audit.event.level`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.stage`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.user_agent`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.request.uri`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.response.code`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.annotation.*`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.object_ref.resource`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.object_ref.name`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.object_ref.namespace`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.object_ref.api_group`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.audit.event.object_ref.api_version`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.user.username`
+|log
+|audit
+|structured metadata
+|
+
+|`k8s.user.groups`
+|log
+|audit
+|structured metadata
+|
+
+|`process.executable.name`
+|resource
+|journal
+|structured metadata
+|
+
+|`process.executable.path`
+|resource
+|journal
+|structured metadata
+|
+
+|`process.command_line`
+|resource
+|journal
+|structured metadata
+|
+
+|`process.pid`
+|resource
+|journal
+|structured metadata
+|
+
+|`service.name`
+|resource
+|journal
+|stream label
+|
+
+|`systemd.t.*`
+|log
+|journal
+|structured metadata
+|
+
+|`systemd.u.*`
+|log
+|journal
+|structured metadata
+|
+|===
+
+[NOTE]
+====
+Attributes marked as *Compatibility attribute* support minimal backward compatibility with the ViaQ data model. These attributes are deprecated and function as a compatibility layer to ensure continued UI functionality. These attributes will remain supported until the Logging UI fully supports the OpenTelemetry counterparts in future releases.
+====
+
+[NOTE]
+====
+Loki changes the attribute names when persisting them to storage. The names will be lowercased, and all characters in the set: (`.`,`/`,`-`) will be replaced by underscores (`_`). For example, `k8s.namespace.name` will become `k8s_namespace_name`.
+====
```

Comment:
We should try to avoid stacking admonitions. See SSG for guidance about multiple admonitions: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
Removed this information as part of Note.